packages/components/cascader/src/cascader.vue

```diff
@@ -48,6 +48,18 @@
           @blur="handleBlur"
           @input="handleInput"
         >
+          <template v-if="$slots.prefix" #prefix>
+            <div
+              style="
```

Comment:
**Question:** why do we use inline styling here? Couldn't we put this into `*.scss` file?